plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -812,6 +909,48 @@ void RemoteVstPlugin::initEditor()
 	// 64-bit versions of Windows use 32-bit handles for interoperability
 	m_windowID = (intptr_t) m_window;
 #endif
+
+#else
+	// native linux support
+	// inspired by https://github.com/ekenberg/vstminihost
```

Comment:
I'm not sure what the license of the original project is, but shouldn't we mention the author information?